### DISCUSSION TITLE
An option for no auth (use third party auth service)

### DISCUSSION THREAD

**Discussion by joshhhhyyyy** [Flexget/webui#162] [Ideas]
Hi,

i am currently migrating my self-hosted stack to cloudflare access.

Would it be realistic to consider a cli option to disable the auth screen you see when you open the site to avoid redundant auth?

**Comment by terribleplan** [comment]
This is a serious issue, especially considering there is a password strength check that prevents you from using the obvious passwords you don't have to remember such as `password`, `admin`, and `flexget`. Whether this is even reasonable for a selfhosted piece of software is another matter (it isn't), but being able to disable the authentication system all together is important when running behind something like cloudflare access or (in my case) oauth2_proxy. If you run it on a local network or VPN network it would be nice to disable it as well.
